projects/plugins/jetpack-boost/LICENSE.txt

```diff
@@ -0,0 +1,357 @@
+This program is free software; you can redistribute it and/or modify
```

Comment:
You can delete this. If you set `.license` to "GPL-2.0-or-later" in composer.json, it'll get copied into the mirror during the build.

Reply:
Deleted https://github.com/Automattic/jetpack/pull/20512/commits/20530e155c219bc6baf3899c5469f03dabbe4025

License was already set.